src/app/customizer/gradient-generator/page.js

```diff
@@ -0,0 +1,302 @@
+"use client";
+import Nav from "@/app/assets/nav";
+import React, { useEffect, useRef, useState } from "react";
+import { FaGithub, FaHome, FaExpandArrowsAlt } from "react-icons/fa";
+import ColorsList from "./components/ColorsList";
+import "./gradient-generator.css";
+import CopyCSSModal from "./components/CopyCSSModal";
+import GradientType from "./components/GradientType";
+import GradientRotation from "./components/GradientRotation";
+import GradientFullScreen from "./components/GradientFullScreen";
+
+const people = [
```

Comment:
Why do we need this object?

Reply:
It was a testing object that I used initially but forgot to remove it at the end.